backend/api/urls.py

```
@@ -0,0 +1,21 @@
+from django.urls import path, include
+from rest_framework.routers import DefaultRouter
+from .views import (
+    RoleViewSet, CustomUserViewSet, CompanyViewSet,
+    PaymentViewSet, CoinTransactionViewSet, SurveyResponseViewSet,
+    VideoViewSet, EmotionViewSet, ShareLinkViewSet, AnalyticsReportViewSet
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)